bins/revme/src/statetest/runner.rs

```diff
@@ -299,7 +333,7 @@ pub fn execute_test_suit(path: &Path, elapsed: &Arc<Mutex<Duration>>) -> Result<
     Ok(())
 }
 
-pub fn run(test_files: Vec<PathBuf>, single_thread: bool) -> Result<(), TestError> {
+pub fn run(test_files: Vec<PathBuf>, single_thread: bool, trace: bool) -> Result<(), TestError> {
```

Comment:
In this function can you add that if `trace` is true we default to `single_thread` true, if it is multithreaded and stdout it will make junk.

Reply:
yes!